doc/boss_tutorial.md

```diff
@@ -0,0 +1,293 @@
+# PaddlePaddle Elastic Deep Learning
+
+<img src="../logo/edl.png" width="500">
+
+While many hardware and software manufacturers are working on improving the training performance
+of deep learning jobs, PaddlePaddle Elastic Deep Learning (EDL) optimize the global utilization
+of the cluster and the waiting time of job submitters. EDL includes two parts, one is a
+Kubernetes controller, PaddlePaddle auto-scaler which changes the number of processes of the 
+distributed jobs according to the idle hardware resource in the cluster,
+and another one is the fault-tolerant distributed training architecture in PaddlePaddle.
+
+## Tutorial Outline
+
+- Introduction
+  At the introduction session, we will introduce:
+    1. a new version named Fluid on PaddlePaddle; and
```

Comment:
demo code written with fluid may not go on to EDL

Reply:
Done.
But we have already intro Fluid in the intro session, maybe we need to let the user know how to use Fluid to train models.